app/models/talentlms/course.rb

```diff
@@ -68,5 +85,14 @@ def completed_by?(user)
 
       Talentlms::CompletedTraining.where(login_id: login.id, course_id: id).exists?
     end
+
+    def active_date_range_for_display
+      return start_date if start_date.present? && end_date.present? && start_date == end_date
```

Comment:
This should be impossible based on the validation that ensures `end_date > start date` (also if needed I think you can just use: `start_date.present? && start_date == end_date`

Reply:
This should be able to occur. The validation should only be firing if both start and end dates are present and start date is > end date. It should be valid to have it start and end on the same date, but I'll double check the logic to be sure.

Reply:
Ah, that makes sense.  Disregard.